### ISSUE TITLE
Installing graphicsmagick and imagemagick for the dmg icon generation in Github Actions  is unbearably slow

### ISSUE THREAD

**Issue by bjesuiter** [sindresorhus/create-dmg#57]
I integrated your create-dmg tool into my github actions workflow: 
https://github.com/bjesuiter/macos-file-summoner/blob/master/.github/workflows/go.yml

However, when installing imagemagic and graphicsmagick, the automated build takes ages longer than without them, because they are so big.

I tried to combat this by using gihub cache actions and caching the homebrew cache dir, but since this thing has 2,4 gb this is also painfully slow. 

Could you implement another solution which is more lightweight to update this dmg icon?

**Comment by bjesuiter** [comment]
Please see this run of the workflow for checking homebrew cache sizes and timings: 
https://github.com/bjesuiter/macos-file-summoner/runs/789695326?check_suite_focus=true

(This run threw an error because of a missing code signing identity, but I simply forgot to load my dev identity there)

**Comment by sindresorhus** [comment]
Yes, it could be implemented as a simple Swift binary using CoreGraphics. However, it’s not something I have time to work on, but pull request welcome.

**Comment by bjesuiter** [comment]
Thanks for looking into the issue! Unfortunately, I don't have any experience with swift, I recently started with this File Summoner App written in go and packaged manually as MacOS App Package.

But maybe this can be my starting point with swift 👍

**Comment by sindresorhus** [comment]
Fixed by https://github.com/sindresorhus/create-dmg/pull/100
